userguide.md

```diff
@@ -0,0 +1,104 @@
+---
+page_title: Docker Swarm
+page_description: Swarm: a Docker-native clustering system
+page_keywords: docker, swarm, clustering
+---
+
+# Docker Swarm: a Docker-native clustering system
+
+Docker `swarm` helps you control a cluster of Docker hosts (known as nodes)
+and expose them as a single "virtual" host.
+
+The Docker `swarm` manager can be interacted with using the Docker API, which means
+any tool which can communicate with a Docker Daemon using that API, can control
+a Docker swarm transparently: dokku, fig, krane, flynn, deis, docker-ui, shipyard,
+drone.io, Jenkins... and of course the Docker client itself.
+
+Like the other Docker projects, `swarm` follows the "batteries included but removable"
+principle. It ships with a simple scheduling backend out of the box. The goal is
+to provide a smooth out-of-box experience for simple use cases, and allow swapping
+in more powerful backends, like `Mesos`, for large scale production deployments.
+
+## Installation
+
+> **Note**: The only requirement for Swarm nodes is they all run the _same_ release
+> Docker daemon (version `1.4.0` and later), configured to listen to a `tcp`
+> port that the Swarm manager can access.
+
+Docker `swarm` is currently only available as a single go binary on Linux. Download
+it from [the latest release](https://github.com/docker/swarm/releases/latest) page
+on GitHub.
+
+For example:
+
+```
+	$ wget -O swarm https://github.com/docker/swarm/releases/download/v0.1.0-rc1/swarm-Linux-x86_64
+	# OR
+	$ curl -SsL https://github.com/docker/swarm/releases/download/v0.1.0-rc1/swarm-Linux-x86_64 > swarm
+	$ chmod 755 swarm
+	$ sudo cp swarm /usr/local/bin
+```
+
+## Nodes setup
+
+Each swarm node will run a swarm node agent which will register the referenced
+Docker daemon, and will then monitor it, updating the discovery backend to its
+status.
+
+The following example uses the Docker Hub based `token` discovery service:
+
+```bash
+# create a cluster
+$ swarm create
+6856663cdefdec325839a4b7e1de38e8 # <- this is your unique <cluster_id>
+
+# For each of your nodes, start a swarm agent
+#  the Docker daemon <node_ip> doesn't have to be public (eg. 192.168.0.X),
+#  as long as the swarm manager can access it.
+$ swarm join --addr=<node_ip:2375> --discovery token://<cluster_id>
```

Comment:
its in the preview release, which is what i'm documenting atm.

will we be getting an rc2 this week?